### PR TITLE
Update Sunder and Rainbow to VI and III respectively, as they are their new highest tiers.

### PR DESCRIPTION
**Pull request by KTrain5169**
Now it's merging to the `master` branch!

### CODE REVIEW

**Review by jani270 — CHANGES_REQUESTED** [review]
,rainbow:2, ,sunder:5,

**Review by KTrain5169 — COMMENTED** [review]
Now fixed

File: items/MAXED_ENCHANT_BOOK.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "§9Smite VII§9, §9Smoldering V§9,",
     "§9Snipe IV§9, §9Spiked Hook VI§9,",
     "§9Strong Mana X§9, §9Sugar Rush III§9,",
-    "§9Sunder V§9, §9Syphon V§9,",
+    "§9 VI§9, §9Syphon V§9,",
```
Comment (jani270) [inline items/MAXED_ENCHANT_BOOK.json:61]: ???

**Review by jani270 — CHANGES_REQUESTED** [review]

File: items/MAXED_ENCHANT_BOOK.json (hunk 1 above)
Comment (KTrain5169) [reply items/MAXED_ENCHANT_BOOK.json:61]: ...

**Comment by KTrain5169** [comment]
> I've detected some problems you might want to take a look at, you can see them as annotations in the files tab.
completely normal, idc

I also double checked the file this time and that should be all of the problems fixed (and most of those were caused by me somehow)

**Comment by CalMWolfs** [comment]
> > I've detected some problems you might want to take a look at, you can see them as annotations in the files tab.
> 
> completely normal, idc
> 
> I also double checked the file this time and that should be all of the problems fixed (and most of those were caused by me somehow)

well its not fixed yet

**Comment by KTrain5169** [comment]
> > > I've detected some problems you might want to take a look at, you can see them as annotations in the files tab.
> > 
> > 
> > completely normal, idc
> > I also double checked the file this time and that should be all of the problems fixed (and most of those were caused by me somehow)
> 
> well its not fixed yet

did I somehow delete the lore again

**Comment by jani270** [comment]
> > > > I've detected some problems you might want to take a look at, you can see them as annotations in the files tab.
> > > 
> > > 
> > > completely normal, idc
> > > I also double checked the file this time and that should be all of the problems fixed (and most of those were caused by me somehow)
> > 
> > 
> > well its not fixed yet
> 
> did I somehow delete the lore again

Indeed

**Comment by jani270** [comment]
> I've detected some problems you might want to take a look at, you can see them as annotations in the files tab.

lol

**Comment by KTrain5169** [comment]
fixed, now I gotta hope I somehow didn't destroy something again. for like the 7th (?) time

**Comment by KTrain5169** [comment]
it somehow took me 8 fixes (including some stuff in `prerelease` branch that somehow I made) and now I have to do it a 9th time do I?

**Review by jani270 — CHANGES_REQUESTED** [review]
,rainbow:2,

**Comment by KTrain5169** [comment]
> ,rainbow:2,

that did not appear when I looked at it, mb

**Comment by KTrain5169** [comment]
I can't spell or click well today

**Review by jani270 — APPROVED** [review]

**Pull request merged**